bundles/org.smarthomej.binding.tuya/src/main/java/org/smarthomej/binding/tuya/internal/local/dto/IRCode.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2021-2023 Contributors to the SmartHome/J project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.smarthomej.binding.tuya.internal.local.dto;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link IRCode} represents the IR code decoded messages sent by Tuya devices
+ *
+ * @author DmitryP. (d51x) - Initial contribution
+ */
+@NonNullByDefault
+public class IRCode {
```

Comment:
```suggestion
public class IrCode {
```

use camel-case where possible